helix-term/src/ui/editor.rs

```diff
@@ -820,6 +823,45 @@ impl EditorView {
         }
     }
 
+    /// Apply the highlighting on the columns where a cursor is active
+    pub fn highlight_cursorcolumn(
+        doc: &Document,
+        view: &View,
+        surface: &mut Surface,
+        theme: &Theme,
+    ) {
+        let text = doc.text().slice(..);
+
+        let primary_style = theme.get("ui.cursorline.primary");
+        let secondary_style = theme.get("ui.cursorline.secondary");
```

Comment:
I'm split on whether these should be specific to cursorcolumn too. We could make them `ui.cursorcolumn.{primary.secondary}` and have them fall back to cursorline themes?